test/mitmproxy/data/har_files/firefox.json

```diff
@@ -174,14 +174,6 @@
                 [
                     "x-amz-cf-id",
                     "DPEkuUbeK1ZXMsRuUHqgk6iE4l7ShgyrJntkqIbLaSJ5646Ptc2Xew=="
-                ],
-                [
-                    "content-type",
-                    "text/plain; charset=utf-8"
-                ],
```

Comment:
We no longer add a `content-type` header if it does not exist in the recorded response.